cypress.json

```
@@ -2,6 +2,7 @@
   "viewportWidth": 1920,
   "viewportHeight": 1080,
   "env": {
-    "host": "https://ordino-daily.caleydoapp.org"
+    "host": "https://ordino.caleydoapp.org/app/",
```

```suggestion
    "host": "https://ordino-daily.caleydoapp.org/app/",
```

Sorry, I was wrong. It must test the daily instance. I'm going to revert the last change.